
package ru.bocharova.tm.api.endpoint;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.2.7
 * 2019-07-23T22:22:32.111+03:00
 * Generated source version: 3.2.7
 */

@WebFault(name = "DataValidateException", targetNamespace = "http://endpoint.api.tm.bocharova.ru/")
public class DataValidateException_Exception extends Exception {

    private ru.bocharova.tm.api.endpoint.DataValidateException dataValidateException;

    public DataValidateException_Exception() {
        super();
    }

    public DataValidateException_Exception(String message) {
        super(message);
    }

    public DataValidateException_Exception(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public DataValidateException_Exception(String message, ru.bocharova.tm.api.endpoint.DataValidateException dataValidateException) {
        super(message);
        this.dataValidateException = dataValidateException;
    }

    public DataValidateException_Exception(String message, ru.bocharova.tm.api.endpoint.DataValidateException dataValidateException, java.lang.Throwable cause) {
        super(message, cause);
        this.dataValidateException = dataValidateException;
    }

    public ru.bocharova.tm.api.endpoint.DataValidateException getFaultInfo() {
        return this.dataValidateException;
    }
}
